email_marketing/mailing_lists.rst

```diff
@@ -0,0 +1,53 @@
+=======================
+Work with Mailing Lists
+=======================
+Mailing lists could be a goldmine for your company’s marketing department as they can provide leads
+for sales, focus groups for testing new products, and consumers to provide feedback on satisfaction,
+for example.
+
+Create Mailing lists
+=====================
+Go to :menuselection:`Mailing lists --> Mailing lists --> Create`.
+Enable the option *Is Public* to make the list visible for users when unsubscribing.
```

Comment:
Was you intention to have a line break here ? If so, you should add `| ` (space included) at the start of both lines. If not, the best practice is to avoid breaking the line in the code to show that your intention is not to have a line break in the render.